### PR TITLE
Adding tests for GET 'module_id'

### PR DESCRIPTION
**Pull request by anarang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+- virtualenvwrapper
+
+### Installation:
+- Install all the dependencies.
+
+``` pip3 install -r requirements.txt ```
+
+- Set the PYTHONPATH to your current working directory
+
+``` export PYTHOPATH=</path/to/your/tests directory> ```
```
Comment (noopurmath3) [inline README.md]: There is a typo in the variable name. It should be PYTHONPATH

File: suites/test_get_modules.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import lemoncheesecake.api as lcc
+from lemoncheesecake.matching import *
+import requests
+from helpers import base
+
+
+SUITE = {
+    "description": "Tests for GET /api/module?module_id=<module_id"
```
Comment (noopurmath3) [inline suites/test_get_modules.py]: > is missing after module_id

Comment (anarang) [reply suites/test_get_modules.py]: @noopurmath3 done

File: README.md (hunk 1 above)
Comment (anarang) [reply README.md]: made the change.

**Pull request merged**